narwhals/dataframe.py

```diff
@@ -2846,6 +2846,27 @@ def filter(
 
         return super().filter(*predicates, **constraints)
 
+    def sink_parquet(self, file: str | Path | BytesIO) -> None:
+        """Evaluate the query in streaming mode and write to a Parquet file.
+
+        This allows streaming results that are larger than RAM to be written to disk.
```

Comment:
This is generally true, but the description seem quite polars specific

Reply:
Perhaps we could keep this quite generic
"""
Write LazyFrame to Parquet file.

This may allow larger-than-RAM datasets to be written to disk.
"""
?